src/persist/src/indexed/mod.rs

```diff
@@ -584,11 +591,11 @@ impl<L: Log, B: Blob> Indexed<L, B> {
         Ok(())
     }
 
-    /// Return true if Pending has at least one pending response.
+    /// Return true if Pending has some pending work.
     pub fn has_pending_responses(&self) -> bool {
```

Comment:
reverted this. my fear was that we would receive a compaction response, but then if we never received any other mutating command, we would never have any pending responses, which means the compaction response would never get processed (because we both need step_interval to have elapsed, and for there to be some pending responses before we call step).

In practice, we'll always have a seal every few seconds and my fear is not going to be an issue.